SPTDataLoaderDemo.xcodeproj/project.pbxproj

```diff
@@ -317,6 +317,7 @@
 					"$(inherited)",
 					include,
 				);
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
```

Comment:
No, as the demo project is using iOS 8.0 and up API. Will not build with Xcode 7.3.